### PR TITLE
Feat-Go: add bin2hex and hex2bin function

### PR DESCRIPTION
**Pull request by YogiPristiawan**

### CODE REVIEW

**Review by elianiva — APPROVED** [review]

File: go/hex_2_bin_test.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+	t.Run("It should return error if given parameter is not even length", func(t *testing.T) {
+		tt := &test{
+			param: "4",
+		}
+		_, err := Hex2Bin(tt.param)
+		if err == nil {
+			t.Errorf("expected error, but got nil")
+		}
+		if err != hex.ErrLength {
```
Comment (aldy505) [inline go/hex_2_bin_test.go:31]:
```suggestion
		if !errors.Is(err, hex.ErrLength) {
```

File: go/hex_2_bin.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	"encoding/hex"
+)
+
+// The hex2bin() function converts a string of hexadecimal values to ASCII characters.
+// Parameter
+// -encoded => hexadecimal value to be converted
+// Return
+// - str => string of ASCII characters
+// - error => error if encoded string contain non hexadecimal character
+
```
Comment (aldy505) [inline go/hex_2_bin.go:13]: remove new line

File: go/bin_2_hex_test.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package pehape
```
Comment (aldy505) [inline go/bin_2_hex_test.go:1]:
```suggestion
package pehape_test
```

File: go/hex_2_bin_test.go [hunk 4]
```diff
@@ -0,0 +1,45 @@
+package pehape
```
Comment (aldy505) [inline go/hex_2_bin_test.go:1]:
```suggestion
package pehape_test
```

**Review by aldy505 — CHANGES_REQUESTED** [review]

**Review by aldy505 — APPROVED** [review]

File: go/hex_2_bin.go (hunk 2 above)
Comment (YogiPristiawan) [reply go/hex_2_bin.go]: done

**Pull request merged**